src/interfaces/stepper-interfaces/DefineCompany/define-company-interface.ts

```diff
@@ -4,5 +4,5 @@ import { CoopTypes } from '@/enums'
 export interface DefineCompanyIF {
   valid: boolean
   cooperativeType: CoopTypes
-  officeAddresses: IncorporationAddressIF | {}
+  officeAddresses: IncorporationAddressIF | object
```

Comment:
Cannot use `{}`.

A better implementation would be to allow "officeAddresses" to be null instead of empty (future cleanup).